alembic/versions/db61c62d9bc0_add_customer_lab_contact.py

```diff
@@ -0,0 +1,27 @@
+"""Add customer lab contact
+
+Revision ID: db61c62d9bc0
+Revises: e853d21feaa0
+Create Date: 2023-10-05 12:53:28.435684
+
+"""
+import sqlalchemy as sa
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "db61c62d9bc0"
+down_revision = "e853d21feaa0"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column(
+        table_name="customer",
+        column=sa.Column("lab_contact_id", sa.INTEGER, sa.ForeignKey("user.id")),
```

Comment:
Wondering nf if NULLABLE is set to True by default? if it is, disregard, otherwise add `NULLABLE = True`

Reply:
Nullable is by default True. Also, I think the deployment would automatically break since there is no default value if this was not the case.